### ISSUE TITLE
MacOS: Upon DisplayCAL exit got error: "TypeError: 'NoneType' object is not callable"

### ISSUE THREAD

**Issue by eoyilmaz** [eoyilmaz/displaycal-py3#331]
Upon DisplayCAL exit the following error messages are printed:

```shell
Exiting DisplayCAL
Ran application exit handlers
Exception ignored in: <function Player.__del__ at 0x137c2c680>
Traceback (most recent call last):
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 132, in __del__
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 248, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/adaptation.py", line 128, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/interface.py", line 159, in delete
TypeError: 'NoneType' object is not callable
Exception ignored in: <function Player.__del__ at 0x137c2c680>
Traceback (most recent call last):
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 132, in __del__
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 248, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/adaptation.py", line 128, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/interface.py", line 159, in delete
TypeError: 'NoneType' object is not callable
Exception ignored in: <function Player.__del__ at 0x137c2c680>
Traceback (most recent call last):
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 132, in __del__
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 248, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/adaptation.py", line 128, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/interface.py", line 159, in delete
TypeError: 'NoneType' object is not callable
Exception ignored in: <function Player.__del__ at 0x137c2c680>
Traceback (most recent call last):
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 132, in __del__
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 248, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/adaptation.py", line 128, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/interface.py", line 159, in delete
TypeError: 'NoneType' object is not callable
Exception ignored in: <function Player.__del__ at 0x137c2c680>
Traceback (most recent call last):
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 132, in __del__
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/player.py", line 248, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/adaptation.py", line 128, in delete
  File "/Users/eoyilmaz/Documents/development/displaycal-py3/.venv/lib/python3.11/site-packages/pyglet/media/drivers/openal/interface.py", line 159, in delete
TypeError: 'NoneType' object is not callable
```

**Comment by eoyilmaz** [comment]
Some of them should be fixed with 5aa0d94ca44e2fe1e44f7add5284154c7a84ebe9

**Comment by eoyilmaz** [comment]
Let's close this...
